packages/api/test/unit/utils.spec.js

```diff
@@ -110,4 +125,46 @@ describe('Utils', () => {
       })
     })
   })
+
+  describe('calculateEpoch()', () => {
+    it('should calculate last epoch', async () => {
+      mock.method(TenderdashRPC, 'getGenesis', async () => ({ genesis_time: new Date(0) }))
```

Comment:
I don't think you even need an mock here. Make your function stateless. Just accept arguments and compute result.
then you can easily check it in the test without even mocking (just by passing different arguments, and makin correct expects)